c/src/ml-api-service-private.h

```diff
@@ -66,6 +78,127 @@ typedef struct
   GAsyncQueue *out_data_queue;
 } _ml_service_query_s;
 
+/**
+ * @brief Structure for ml_remote_service
+ */
+typedef struct
+{
+  nns_edge_h edge_h;
+  nns_edge_node_type_e node_type;
+} _ml_remote_service_s;
+
+/**
+ * @brief Creates ml remote service handle with given ml-option handle.
+ * @details The caller should set one of "remote_sender" and "remote_receiver" as a service type in @a ml_option.
+ * @remarks The @a handle should be destroyed using ml_service_destroy().
+ * @param[in] option The option used for creating query service.
+ * @param[out] handle Newly created query service handle is returned.
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
```

Comment:
Q: I need to familiarize myself with the Tizen doxygen; thus have a question here. Is line 96 a summary of the return values declaration, and are the following lines for the details?

Reply:
The order is probably as follows. @brief, @details, @since_tizen, @remarks, @param, @return, @retval, @pre, 
 @post and @see

Reply:
From Tizen native API reference guide, `return` and `retval` are mandatory tags.
```
...
  * @return          [Mandatory] Return values
  * @retval          [Mandatory] Meaning of each return value
...
```